docs/javascript/v2/how-to-guides/set-up-video-conferencing/screen-share.mdx

```diff
@@ -52,6 +52,27 @@ To stop the screenshare:
 hmsActions.setScreenShareEnabled(false);
 ```
 
+## Config for setScreenShareEnabled
+
+The `setScreenShareEnabled` function accepts a second optional config prop. It has the following options:
+
+<br />
+
+-   `videoOnly`: Share only video. Default value is `false`.
+-   `audioOnly`: Share only audio. Default value is `false`.
+-   `forceCurrentTab`: Show the current tab first in the browser prompt and throw an error if any other tab is selected. Default value is `false`.
+-   `preferCurrentTab`: Show the current tab first in the browser prompt but does not throw an error if any other tab is selected. Default value is `false`.
+
+-   `selfBrowserSurface`: Whether to show an option for sharing the current tab in the screen share prompt. Screen sharing current tab might lead to hall of mirrors effect.
+    Default is `exclude`, if either of `forceCurrentTab` or `preferCurrentTab` are `true`, this is set to `include`.
+
+-   `surfaceSwitching`: Include the option to switch tabs while sharing. Default is `include`, but will be set to `exclude` if `forceCurrentTab` is `true`.
+
+-   `systemAudio`: Whether to show option for sharing system level audio if the entire screen
+    is being shared. Possible values are `include` and `exclude`. Not applicable if `videoOnly` is `true`. Note that sharing system audio will cause echo if the mic is on. The default value is `exclude`.
```

Comment:
do we know for sure it would cause echo? if not, let's say might cause echo.

Reply:
Not sure, was an existing comment in the codebase. Will update to say `might`

Reply:
do check this once, if you find a windows device.